bigflow/resources.py

```diff
@@ -174,9 +166,13 @@ def create_setup_body(project_name: str) -> str:
 '''
 
 
+@deprecated(
+    reason="""
+        This method doesn't work when called from pip-installed package.
+        Use `bigflow.build.reflect.materialize_setuppy` instead"""
```

Comment:
I think we can automatically set the beam `options.setup_file = ...` option inside the BeamJob (of course in a separate issue/pull request, in the next release).